industry_benchmarks/utils/results_cleanup.py

```diff
@@ -0,0 +1,349 @@
+            # this means we can grab the first that matches since there is only
+            # one ProtocolUnitResult-*
+            proto_key = next(
+                k
+                for k in results["unit_results"].keys()
+                if k.startswith("ProtocolUnitResult")
+            )
+
+            results_dir = (
+                Path(results["unit_results"][proto_key]["outputs"]["nc"]["path"])
+                .resolve()
+                .parent
+            )
+            # if the dir doesn't exist, we should try and fix it
+            if not results_dir.is_dir():
+                print("Fixing path to results dir")
+                results_dir = remove_first_reversed_sequential_duplicate_from_path(
+                    results_dir
+                )
+
+                # Now we should check if the dir exists
+                if not results_dir.is_dir():
+                    print("Can't find results directory, skipping")
+                    continue
+
+            structural_analysis_data = results["unit_results"][proto_key]["outputs"][
+                "structural_analysis"
+            ]
+
+            # save structural analysis data
+            print("Saving structural analysis data")
+            np.savez_compressed(
+                results_dir / "structural_analysis_data.npz",
+                protein_RMSD=np.asarray(
+                    structural_analysis_data["protein_RMSD"], dtype=np.float32
+                ),
+                ligand_RMSD=np.asarray(
+                    structural_analysis_data["ligand_RMSD"], dtype=np.float32
+                ),
+                ligand_wander=np.asarray(
+                    structural_analysis_data["ligand_wander"], dtype=np.float32
+                ),
+                protein_2D_RMSD=np.asarray(
+                    structural_analysis_data["protein_2D_RMSD"], dtype=np.float32
+                ),
+                time_ps=np.asarray(
+                    structural_analysis_data["time(ps)"], dtype=np.int32
+                ),
+            )
+
+            # remove structural_analysis data stuffed into unit results
+            del results["unit_results"][proto_key]["outputs"]["structural_analysis"]
+
+            # Now we sub sample the traj and save reporter data
+            simulation = results_dir / "simulation.nc"
+            checkpoint = results_dir / "checkpoint.chk"
+            hybrid_pdb = results_dir / "hybrid_system.pdb"
+            # TODO better name?
+            outfile = results_dir / "energy_replica_state.npz"
+            out_traj = results_dir / "out"
+            print("Subsampling trajectory and saving energy data")
+            extract_data(simulation, checkpoint, hybrid_pdb, outfile, out_traj)
+
+            # Now we delete files we don't need anymore
+            print("Deleting trajectory files")
+            os.remove(simulation)
+            os.remove(checkpoint)
+            os.remove(results_dir / "structural_analysis.json")
```

Comment:
I wasn't even aware that we were creating this file 🤦🏽

Reply:
haha we are BUT it is json and not npz, I will make a PR to upstream this change